sdks/java/build.gradle

```diff
@@ -21,6 +9,21 @@ buildscript {
     }
 }
 
+plugins {
+    id 'com.vanniktech.maven.publish' version '0.18.0'
+}
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'com.diffplug.spotless'
+apply plugin: 'signing'
+
+group = 'com.dropbox.sign'
+archivesBaseName = 'dropbox-sign'
+version = '1.0.0-beta2301'
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
+
```

Comment:
Set your snapshot host as well to S01

```
mavenPublish {
  sonatypeHost = S01
}
```

https://github.com/dropbox/dependency-guard/blob/d758d06ff46a4b536067b4a379754a08cba8b5d7/dependency-guard/build.gradle.kts#L58-L60

Reply:
Once the snapshot is published it'll show up at https://s01.oss.sonatype.org/content/repositories/snapshots/com/dropbox/